src/reader/components/ReaderView.vue

```diff
@@ -50,6 +51,7 @@
           WordListWidget,
           TokenAnnotationWidget,
           NewAlexandriaWidget,
+          NamedEntitiesWidget,
```

Comment:
I know there was discussion on this previously, but I think it might be good to have the annotation widget and named entities widgets higher in the list; higher position means it is more likely the widgets will be in the "default" viewport for a user and it will be clearer that selecting a token updates the widget contents (unlike `WordListWidget` and `NewAlexandriaWidget` which do not respond to token selections)